.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Force Unix line endings
+*.js text eol=lf
+*.jsx text eol=lf
+*.css text eol=lf
+*.html text eol=lf
+*.json text eol=lf
+*.yml text eol=lf
```

Comment:
Are there any extensions where we do NOT want eol=lf?

If not, I think this file we can just do
```
* text eol=lf
```

Reply:
@jwu910 I couldn't find a reason, except binaries.  Will this affect binaries?  Do you know of any reasons?

Reply:
@jwu910 I couldn't find a reason, except binaries.  Will this affect binaries?  Do you know of any reasons?

Reply:
Good point

[This issue](https://github.com/jhipster/generator-jhipster/issues/5456) ran into a similar problem

If eslint rule sets to check LF endings, is it too much to just have the .editorconfig specify LF line endings rather than setup the .gitattributes? The linter will warn, and anyone using .editorconfig will auto set the line endings before it makes it to git.

What do you think?